### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by awais-amjed**
In this Pull Request, the following changes were introduced:

- Added Species Column to the animals' table. ✅ 
- Added more data to the table. ✅ 
- Added the required queries to queries.sql. ✅

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOL,
     weight_kg DECIMAL,
+    species TEXT,
```
Comment (VICTOR-CHUKWU) [inline schema.sql]: - [ ] column specie should be a type of `varcahar limit` i.e `varcah(200), kindly fix

File: data.sql [hunk 2]
```diff
@@ -4,3 +4,11 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
...
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', '2021-01-07', '15.04', FALSE, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Devimon', '2017-05-12', '11', TRUE, 5);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Charmander', '2020-02-08', '-11', FALSE, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Plantmon', '2021-11-15', '-5.7', TRUE, 2);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Squirtle', '1993-04-02', '-12.13', FALSE, 3);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Angemon', '2005-06-12', '-45', TRUE, 7);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Boarmon', '2005-06-07', '20.4', TRUE, 7);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', '17', TRUE, 3);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Ditto', '2022-05-14', '22', TRUE, 4);
```
Comment (VICTOR-CHUKWU) [inline data.sql:14]: - [ ] acording to the project requirement, Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction. however no transaction has been done here, kindly fix
:hint 
```BEGIN;

UPDATE animals SET species = 'unspecified';

ROLLBACK;

BEGIN;
UPDATE animals SET species ='digimon'  WHERE name LIKE '%mon';

UPDATE animals SET species ='pokemon'  WHERE species IS NULL;

COMMIT;
```
also you have more than two of such transactions to do kindly check[ project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) for more details

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @awais-amjed 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (awais-amjed) [reply data.sql:14]: I thought we just had to do this in our database and not submit it. I will add those queries.

File: schema.sql (hunk 1 above)
Comment (awais-amjed) [reply schema.sql]: @VICTOR-CHUKWU Thanks for the response. But we can't know that it has a limit of 200 characters? Can you please guide me on how we can decide on the maximum number of characters for a column?

**Review by uwadonat — APPROVED** [review]
Hi **@[awais-amjed](https://github.com/awais-amjed),** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
cong


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**